roles/fairfood/templates/post-receive.j2

```diff
@@ -16,50 +16,62 @@ APP_ROOT="$(pwd)"
 export RAILS_ENV="{{ rails_env }}"
 
 upgrade_application() {
-        local action="${1:-reload}"
+  local action="${1:-reload}"
 
-        source script/rbenv-init.sh
-        rbenv install --skip-existing
-        ./script/install-bundler
+  echo "RAILS_ENV=$RAILS_ENV"
 
-        # Install the required gems
-        # Note: the 'LANG=en_US.UTF-8' is a fix for jquery-rails 1.0.17...fails even though server has correct locale setting
-        LANG="en_US.UTF-8" bundle --path "$HOME/.gem" --deployment --without "development" "test"
+  source script/rbenv-init.sh
+  ./script/rbenv-install.sh
+  ./script/install-bundler
 
-        ./script/precompile-assets
+  # Install the required gems
+  bundle --path "$HOME/.gem" --deployment --without "development" "test" | egrep -v '^Using.*$'
```

Comment:
`grep` would have worked, too. `egrep` is needed if you use something like `Using.+` instead of `*`. But looking at the regex, you could just do this:
```suggestion
  bundle --path "$HOME/.gem" --deployment --without "development" "test" | grep -v '^Using'
```

Sometimes the smallest thing that doesn't actually matter attracts a lot of attention. Probably a case of bikeshedding. :wink:

Reply:
Ah, yes good one. I just default to regex where I have full control ;)